static/src/javascripts/projects/common/modules/experiments/tests/remote-epic-variants.js

```diff
@@ -15,23 +13,15 @@ const remoteVariant: Variant = {
 export const remoteEpicVariants: Runnable<AcquisitionsABTest> = {
     id,
     start: '2020-05-01',
-    expiry: '2020-09-06',
+    expiry: '2020-09-08',
     author: "Nicolas Long",
     description: "Pseudo-test to use remote service for % of contribution epics. Expected to run as highest priority test; the canRun will then narrow the audience.",
     audience: 1,
     audienceOffset: 0,
     successMeasure: "Revenue/impressions equivalent to local variants",
     audienceCriteria: "All",
     variants: [remoteVariant],
-    canRun: () => {
-        // Delay geolocation due to known race condition
-        // https://github.com/guardian/frontend/pull/22322
-        const geolocation = geolocationGetSync();
-        return config.get("switches.abRemoteEpicVariants") && geolocation !== 'AU' && Math.random() < 0.01// set test % here
```

Comment:
do you want to keep the switch?

Reply:
I don't think it is actually needed - the Frontend AB code already checks this.